packages/fscomponents/src/components/UpdateNameOrEmail.tsx

```diff
@@ -1,27 +1,34 @@
 import React, { Component } from 'react';
-import { View } from 'react-native';
+import {
+  NativeSyntheticEvent,
+  StyleProp,
+  TextInputFocusEventData,
+  View,
+  ViewStyle
+} from 'react-native';
 
 // @ts-ignore TODO: Update tcomb-form-native to support typing
 import * as t from 'tcomb-form-native';
 import { emailRegex } from '../lib/email';
 import { Form } from './Form';
 import { Button } from './Button';
 import FSI18n, { translationKeys } from '@brandingbrand/fsi18n';
+import {Dictionary} from '@brandingbrand/fsfoundation';
 const componentTranslationKeys = translationKeys.flagship.updateNameOrEmail;
 
 export interface UpdateNameOrEmailState {
   value: any;
 }
 
 export interface UpdateNameOrEmailProps {
-  fieldsStyleConfig?: any; // the custom stylesheet we want to merge with the default stylesheet
-  onSubmit?: (value: any) => void; // the behaviour we want onpress of submit button
-  submitButtonStyle?: any;
-  submitTextStyle?: any;
-  submitText?: any; // Text to override the submit button
-  style?: any;
-  fieldsOptions?: any; // any extra desired behaviour, like placeholders
-  value?: any;
+  fieldsStyleConfig?: Dictionary; // custom stylesheet we want to merge with the default stylesheet
+  onSubmit?: <T>(value: T) => void; // the behaviour we want onpress of submit button
```

Comment:
Please make this `<T = any>` to preserve backwards compatibility.

Reply:
@bweissbart Resolved.